protos/dcs/common/common.proto

```diff
@@ -349,3 +349,14 @@ message MarkPanel {
   optional string text = 6;
   Position position = 7;
 }
+
+/**
+ * A vector in a left-handed coordinate system with +z being north, -z south, +x east, -x west,
+ * +y up and -y down (as opposed to DCS' unusual right-handed coordinate system where x is north/
+ * south and z is west/east).
+ */
+ message Vector {
+  double x = 1;
+  double y = 2;
+  double z = 3;
+}
```

Comment:
I am still not 100% sure whether it is a good idea to change the coordinate system, but we are also changing the wind from a TO direction to a FROM direction - so it is at least not new that we are second guessing DCS.

Pros:
- More common, easier to apply knowledge / code-snippets from general game dev tutorials.

Cons:
- Possibly harder to use for people coming from a DCS Lua background (even though I think that only few people dived into 3D math using DCS vectors).

If the gRPC server wouldn't use the coordinate system as used in this PR, I'd convert vectors on the client-side, which wouldn't be a big deal too.

Reply:
I am fine changing the coordinate system at the gRPC boundary. This is sort of an extension of our decision a wile ago not to be slaves to the ED APIs.

However I would extensively document this to make it clear that we are doing it and what the values would be in the lua scripting environment. (e.g. y = z or whatever).

Reply:
Extended the comment slightly, let me know if you'd like to see more explanations